packages/flame/lib/src/events/messages/drag_update_event.dart

```diff
@@ -36,4 +36,10 @@ class DragUpdateEvent extends PositionEvent {
       ),
     );
   }
+
+  @override
+  String toString() =>
+      'DragUpdateEvent(pointerId: $pointerId, timestamp: $timestamp, '
```

Comment:
```suggestion
      '$DragUpdateEvent(pointerId: $pointerId, timestamp: $timestamp, '
```
Will help in keeping class name in this string in-sync with the actual class name in future as well.